grails-app/services/org/pih/warehouse/requisition/RequisitionService.groovy

```diff
@@ -891,6 +895,10 @@ class RequisitionService {
 
     void deleteComment(Comment comment, Requisition requisition) {
         requisition.removeFromComments(comment)
+        Event event = Event.findByComment(comment)
+        if (event) {
+            event.comment = null
```

Comment:
Will the comment be deleted at this point? I assume it won't, because you don't have cascade/orphan delete on `comment` on the `Event` side, do you?

Reply:
but I do on Requisition so `requisition.removeFromComments(comment)` should handle the delete.